src/azure-cli/azure/cli/command_modules/acs/_client_factory.py

```diff
@@ -64,7 +64,7 @@ def get_container_service_client(cli_ctx, **_):
 def get_osa_container_service_client(cli_ctx, **_):
     from azure.mgmt.containerservice import ContainerServiceClient
 
-    return get_mgmt_service_client(cli_ctx, ContainerServiceClient, api_version='2019-04-30')
+    return get_mgmt_service_client(cli_ctx, ContainerServiceClient, api_version='2019-09-30-preview')
```

Comment:
Is this what we want? I was under the impression that we don't want to use the new API version for the CLI unless customers are using the new features (e.g. Azure Monitor). cc @amanohar

Reply:
As discussed on our Azure Monitor meeting today, we are okay with moving all the AZ CLI functionality to the new preview API, as multiple versions are not supported.